document_scanner_webinar/README.MD

```diff
@@ -0,0 +1,48 @@
+# Document Scanner Component
+
+![Photo of the Document Scanner](screenshots/documentScanner.png)
+
+This is a Custom Component for Superblocks, that uses [React-Webcam](https://github.com/mozmorris/react-webcam) to bring image capture inside the Superblocks frontend.
+
+Getting started
+=====
+
+### Requirements
+
+To deploy this project you'll need accounts for the following services:
+
+- [Superblocks](https://superblocks.com/)
+
+### Setup
```

Comment:
I'd rather this be on the main README or we just point to the docs. Feels weird to have it inside this specific example

Reply:
It should be in all of them. You never know where SEO is gonna send you.